Factory/Classes/Facilities/Warehouse.m

```diff
@@ -77,8 +86,8 @@ - (NSSet *)shipWaresOfCount:(NSUInteger)count
         for (id<WareProtocol> ware in mutableShipment) {
             [self.wares removeObjectForKey:[ware uniqueIdentifier]];
         }
-
-        return [mutableShipment copy];
+        
+        return [mutableShipment autorelease];
```

Comment:
This is not the fix I'd expected to see; I still want to return an immutable set.